optiga-trust-x/Kconfig

```diff
@@ -0,0 +1,64 @@
+# Options for Optiga Trust X security chip 
```

Comment:
Please move the Kconfig file to the main zephyr, tree, under this folder:
https://github.com/zephyrproject-rtos/zephyr/tree/master/modules

Reply:
Should i do PR with that Kconfig file right now or should wait until proper **not draft** PR. Then with one PR it will be possible to add Kconfig file and update west.yml with module information [like specified here](https://docs.zephyrproject.org/latest/guides/modules.html#process).